Build failure with GHC 9.0.1

```
src/Commonmark/Inlines.hs:333:10: error:
    A pragma is not allowed in this position: {-# SCC attrParser #-}
    |
333 |          {-# SCC attrParser #-} (AddAttributes <$> attrParser)
    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Commonmark/Inlines.hs:337:7: error:
    A pragma is not allowed in this position: {-# SCC pDelimChunk #-}
    |
337 |   <|> {-# SCC pDelimChunk #-} pDelimChunk specmap isDelimChar
    |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

I can't find anything about a relevant parser change in GHC's [release notes](http://downloads.haskell.org/~ghc/latest/docs/html/users_guide/9.0.1-notes.html) or the [migration guide](https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0), so I'm wondering whether this could be a regression.

--- a/commonmark/src/Commonmark/Inlines.hs
+++ b/commonmark/src/Commonmark/Inlines.hs
@@ -335,11 +335,11 @@ pChunk :: (IsInline a, Monad m)
 pChunk specmap attrParser ilParsers isDelimChar =
  do pos <- getPosition
     (res, ts) <- withRaw $
-         {-# SCC attrParser #-} (AddAttributes <$> attrParser)
+         ({-# SCC attrParser #-} AddAttributes <$> attrParser)
          <|>
          {-# SCC pInline #-} (Parsed <$> pInline ilParsers)
     return $! Chunk res pos ts
-  <|> {-# SCC pDelimChunk #-} pDelimChunk specmap isDelimChar
+  <|> ({-# SCC pDelimChunk #-} pDelimChunk specmap isDelimChar)
   <|> (do t <- anyTok
           endpos <- getPosition
           return $! Chunk

